tests/integration/cli/tests/config.rs

```diff
@@ -0,0 +1,196 @@
+use anyhow::bail;
+use std::path::{Path, PathBuf};
+use std::process::Command;
+use wasmer_integration_tests_cli::get_wasmer_path;
+
+#[test]
+fn config_works() -> anyhow::Result<()> {
+    let bindir = Command::new(get_wasmer_path())
+        .arg("config")
+        .arg("--bindir")
+        .output()?;
+
+    let bin_path = Path::new(env!("WASMER_DIR")).join("bin");
+    assert_eq!(
+        String::from_utf8(bindir.stdout).unwrap(),
+        format!("{}\n", bin_path.display())
+    );
+
+    let bindir = Command::new(get_wasmer_path())
+        .arg("config")
+        .arg("--cflags")
+        .output()?;
+
+    let include_path = Path::new(env!("WASMER_DIR")).join("include");
+    assert_eq!(
+        String::from_utf8(bindir.stdout).unwrap(),
+        format!("-I{}\n", include_path.display())
+    );
+
+    let bindir = Command::new(get_wasmer_path())
+        .arg("config")
+        .arg("--includedir")
+        .output()?;
+
+    let include_path = Path::new(env!("WASMER_DIR")).join("include");
+    assert_eq!(
+        String::from_utf8(bindir.stdout).unwrap(),
+        format!("{}\n", include_path.display())
+    );
+
+    let bindir = Command::new(get_wasmer_path())
+        .arg("config")
+        .arg("--libdir")
+        .output()?;
+
+    let lib_path = Path::new(env!("WASMER_DIR")).join("lib");
+    assert_eq!(
+        String::from_utf8(bindir.stdout).unwrap(),
+        format!("{}\n", lib_path.display())
+    );
+
+    let bindir = Command::new(get_wasmer_path())
+        .arg("config")
+        .arg("--libs")
+        .output()?;
+
+    let lib_path = Path::new(env!("WASMER_DIR")).join("lib");
+    assert_eq!(
+        String::from_utf8(bindir.stdout).unwrap(),
+        format!("-L{} -lwasmer\n", lib_path.display())
+    );
+
```

Comment:
Can you test two flags at once?

Right now:

```bash
$ wasmer config --bindir --cflags
/Users/syrusakbary/.wasmer/bin
-I/Users/syrusakbary/.wasmer/include
```

But `--pkg-config` should not be used with other flags together:

```bash
$ wasmer config --bindir --cflags --pkg-config
error: The argument '--bindir' cannot be used with '--pkg-config'

USAGE:
    wasmer config --bindir --cflags

For more information try --help
```

Reply:
Implemented in https://github.com/wasmerio/wasmer/pull/3428/commits/e96c31318b56f7648271b702fe272da8d3a47327 and https://github.com/wasmerio/wasmer/pull/3428/commits/ca50d8218c19da7485014544606d187523ac8815